clouddriver-kubernetes-v2/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesManifestStrategyTest.java

```diff
@@ -288,6 +291,35 @@ void useSourceCapacityToAnnotations() {
         .containsOnly(entry("strategy.spinnaker.io/use-source-capacity", "true"));
   }
 
+  @ParameterizedTest
+  @EnumSource(DeployStrategy.class)
```

Comment:
Wow this is amazing, I can't wait to use this!

Reply:
I've known about this for 90 minutes and already used it 3 times!